src/Copy/Text.elm

```diff
@@ -432,40 +432,40 @@ t key =
             "People you can talk to"
 
         GetHelpSection1Title ->
-            "Join the SEA survivors’ forum"
+            "Support from survivors"
 
-        GetHelpSection1Quote1 ->
+        GetHelpSection1Quote ->
             "Meeting other people who’ve been through what I’ve been through is such a massive help. It’s amazing to know people who can really understand what I went through."
```

Comment:
Do we want to transform these to `'`?

Reply:
Yes. I thought I'd got them all. It might be in the other PR.

Reply:
Looks to be - maybe #32 should go in first, I've approved that.